src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.c

```diff
@@ -668,9 +684,23 @@ static bool pick_from_internal_rr_locked(
 
 static grpc_lb_policy_args *lb_policy_args_create(grpc_exec_ctx *exec_ctx,
                                                   glb_lb_policy *glb_policy) {
-  grpc_lb_addresses *addresses =
-      process_serverlist_locked(exec_ctx, glb_policy->serverlist);
+  GPR_ASSERT(glb_policy->serverlist != NULL || 
+             glb_policy->fallback_backend_addresses != NULL);
+  grpc_lb_addresses *addresses;
+  if (glb_policy->serverlist != NULL) {
+    GPR_ASSERT(glb_policy->serverlist->num_servers > 0);
+    addresses = process_serverlist_locked(exec_ctx, glb_policy->serverlist);
+  } else if (glb_policy->fallback_backend_addresses != NULL) {
```

Comment:
Note that in order to make this work, we need to make sure that `fallback_backend_addresses` is never set to NULL until we get the first response from the balancer.  Otherwise, we wind up triggering an assertion somewhere, because we have no addresses to pass to the RR policy.

See my comment in lb_policy_factory.c.